### ISSUE TITLE
Update Landing Page Content

### ISSUE THREAD

**Issue by Dianadec** [bcgov/cas-registration#163]
#### Describe the task

This card is meant to contain the approved content from the business area of the text/info to go onto the landing page of the app.

Please refer to the [design](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=2688-6375&mode=design&t=y7smsRJhRtzkRitv-4) in Figma.

#### Acceptance Criteria

- [ ] Meta tag for description has text = TBD
- [ ] Content is responsive for mobile and desktop
- [ ] Jest unit test (with snapshot?)
- [ ] e2e tests

#### Additional context

- Blocked by #112 
- Blocked by #260

**Comment by suhafa** [comment]
Is this ticket blocked by #260 as well where we will get clear description of the content from business area?

**Comment by suhafa** [comment]
For now, we will have one landing page for both external and internal users. Later we will have a separate link

**Comment by Dianadec** [comment]
@suhafa  do we have updated info that we could add to this card to refine it?

**Comment by suhafa** [comment]
@Dianadec currently this is blocked by progress on ticket #260

**Comment by hannavovk** [comment]
Dec 4 update: We have a reasonably close draft page, but it doesn't look finalized enough. Working with BA to finalize.

**Comment by hannavovk** [comment]
@nanyangpro, would you mind updating this card with the current info. I think it's ready to go for devs.

**Comment by nanyangpro** [comment]
Sure thing, thanks @hannavovk!

Done. Feel free to remove the "Backlog Refinement" label unless the developers want to add in more AC, thanks @Dianadec!

**Comment by hannavovk** [comment]
Nice to see the super old placeholder landing page gone!
I see the Program Admin and Industrial Operator log in buttons are still in the banner - will they be removed later or should it be part of this ticket? 

![Image](https://github.com/bcgov/cas-registration/assets/87211104/2b458d54-95f2-425a-a3e1-c8a2b49d2a64)

Also, would it be terribly annoying if I asked for some modifications to improve the styling of the page? That would be both design and app updates. Just thinking of making a good first impression :). I don't mind being told otherwise though. 

cc @marcellmueller @nanyangpro @Dianadec

**Comment by marcellmueller** [comment]
Sorry I just redid the landing page content and didn't touch the nav. Though it is super easy to remove the buttons. 

We definitely should have updated the designs before I completed this though I'm sure we can fit in some updates especially if they aren't too big. Can you please talk to Nan about the changes and make a new ticket with updated designs in Figma? I will try to get to it before launch thanks :)

**Comment by Dianadec** [comment]
@hannavovk are you kind of thinking something more like the CIIP landing page? https://ciip.gov.bc.ca/
I understand what you mean. Is it a font size issue? When I zoom out to like 75% it looks more like the wireframe. @marcellmueller  open to your thoughts too! We can chat during standup. cc @nanyangpro

**Comment by marcellmueller** [comment]
The font size is 24px in the wireframe and on the page itself aside from the Dates are which is smaller.

**Comment by nanyangpro** [comment]
The design is easy to change (did it exactly as the IRC version, not super satisfied too), let's chat! @hannavovk @Dianadec 
Agreed with @marcellmueller, dealing with additional changes with the design updated first would be ideal.
